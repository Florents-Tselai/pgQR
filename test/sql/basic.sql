select qr_ascii('hello world');
select qr('pg_qr', '/tmp/pg_qr.png');
