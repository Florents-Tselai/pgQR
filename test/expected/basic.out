select qr_ascii('hello world');
                              qr_ascii                              
--------------------------------------------------------------------
                                                                   +
                                                                   +
                                                                   +
                                                                   +
         ##############  ##  ##  ##      ##  ##############        +
         ##          ##  ####      ######    ##          ##        +
         ##  ######  ##  ##  ##  ####  ####  ##  ######  ##        +
         ##  ######  ##        ####  ####    ##  ######  ##        +
         ##  ######  ##    ##      ####  ##  ##  ######  ##        +
         ##          ##  ##    ##    ######  ##          ##        +
         ##############  ##  ##  ##  ##  ##  ##############        +
                         ####    ##    ##                          +
             ######  ##  ######  ##  ############    ######        +
         ######    ##    ##      ####  ##      ##    ##            +
         ##      ##  ########        ####  ##    ####  ####        +
         ####  ##  ##  ####      ##      ##  ##        ####        +
             ##    ######  ##  ##########  ################        +
         ##  ######      ##    ####    ####    ##    ##            +
         ##          ##    ##  ##          ##########  ####        +
         ##  ######          ##      ##  ##  ######      ##        +
         ##  ##    ######      ####  ##  ##############            +
                         ##        ####  ##      ##  ##            +
         ##############            ########  ##  ##  ######        +
         ##          ##        ##      ####      ####  ##          +
         ##  ######  ##  ##  ##      ##################            +
         ##  ######  ##  ####      ####      ##  ####    ##        +
         ##  ######  ##  ####  ##########    ####  ##    ##        +
         ##          ##    ######    ####  ##  ####      ##        +
         ##############        ##    ##  ##    ##    ######        +
                                                                   +
                                                                   +
                                                                   +
                                                                   +
 
(1 row)

select qr('pg_qr', '/tmp/pg_qr.png');
 qr 
----
 t
(1 row)

